### ISSUE TITLE
Airbus sharing.

### ISSUE THREAD

**Issue by rafistolique** [Sequal32/yourcontrols#55]
Hello to you,

First, please excuse me for my poor English.
I risk posting a message, because I don't know where to post it elsewhere. Like many I also installed "Your controls". I had to open port 7777, but I didn't encounter any major difficulty.

It works relatively well with all airplanes EXCEPT the airbus. But when we try the Airbus my brother and I, the song is not the same. When we use a button, the other player does not notice it. The lighting works (the "dome" switch not every time). Both players can change their buttons, but this is ignored by the other player, as "your controls" usually do with other planes.

The MCDU is not shared, the altimeter / heading / VS / Speed ​​/ baro, are not shared. Basically the lamps work, two or three buttons to start the aircraft (but not the battery switches, not the "engine one and two" switches). Ecam and its interupteurs are not shared either ... Except, on youtube, I see players succeed in sharing the mcdu and others.
 We can't do it, whereas with the other planes, there is no problem.

Do you have any idea what to do? We tested with FlyByWire A32 in "dev" or "stable" version, with or without update, we took the last update of "your controls" from yesterday, but that does not change anything.

Thank you for reading this post.

**Comment by Sequal32** [comment]
Hi! I believe this issue was caused by the latest MSFS update. I recommend you delete the YourControls folder from the community package location, and reinstall using the installer. Make sure you install YourControls *after* you install the A32NX.

**Comment by rafistolique** [comment]
Thank you very much for taking the time to answer me. When I get home from work tonight, I'll test this out without waiting any longer. Again last night we flew a Cessna (g1000) for more than two hours, without ANY problem. And I must admit that I am really delighted with the current result. I will come and post the verdict and thank you again for this wonderful work.

**Comment by rafistolique** [comment]
Hello, as expected here is my test return. Indeed the solution was there, it was enough to reinstall "your controls". I'm ashamed that I didn't think about it. It finally works. Thank you for your time and apologize for the inconvenience.

THANK YOU

**Comment by larsf96** [comment]
Hello, 
I'm experiencing the same problem as @rafistolique. I've also reinstalled YourControl, but the issue still exists. Everything worked fine before the latest MSFS update, so maybe that update broke something.
Any idea how to debug the problem?

**Comment by rafistolique** [comment]
Hi to you,

For our part, my brother and I deleted the "your controls" directory from the "community" folder. We also, my brother and I, erased the contents of the "AppData" directory. We no longer wanted a trace of "your controls".
Then we reinstalled the DEV version of "A32NX Fly By Wire", (The folder already existed in community, but we just clicked on "install the" dev "mode).
During our last tests, we stopped at a stable version.

Once done, we launched the "installer" file of "Your controls", chosen the "DEV" version. And this app again placed the "Your controls" folder in community.

Once "your controls" started, we had to retype the name, the host IP, the choice of the plane (here Airbus in DEV mode). The proof that there was no longer any trace of the old installation.

At this point we had a problem, the host had a "script has stopped" message and the client had their program exiting. But the client (my brother) had chosen the wrong plane. Once the host and the client chose the same plane, there were no more scripting problems, and we were able to play together without "problems".

(My brother had a "community" folder super loaded with mods of all kinds. I only had two scenes. He moved all his addons except the two scenes we had in common. I don't think that has changed. many things).

Unlike the other planes, here we see that if the autopilot is engaged for a long time, the client obtains a bad frame rate (it suffices to temporarily cut off the autopilot and the frame rate improves), but this is another topic.

I hope I have been as detailed as possible.

Good luck to you.
